R/database.R

```diff
@@ -2698,19 +2703,24 @@ get_manta_sv_by_samples = function(these_samples_metadata,
 #' @export
 #'
 #' @examples
-#' my_svs = get_manta_sv_by_sample(this_sample_id = "00-14595_tumorD",
-#'                                 these_samples_metadata = get_gambl_metadata())
-#'
+#' #example 1
+#' #get manta calls for a sample that needs to be lifted to "hg38" and let this function take care of the liftover step for you. 
+#' my_sv = get_manta_sv_by_sample(this_sample_id = "99-27783_tumorA", these_samples_metadata = get_gambl_metadata(), projection = "hg38", force_lift = TRUE)
```

Comment:
Suggestion for Example 2, assuming this works. It shows how the user can either provide just the metadata for their sample of interest or use the combination of parameters.

```
my_metadata = get_gambl_metadata() %>% dplyr::filter(sample_id=="99-27783_tumorA")
my_sv = get_manta_sv_by_sample(these_samples_metadata = my_metadata, projection = "hg38", force_lift = TRUE)
```

Reply:
Added!